rpm/help.md

```diff
@@ -90,6 +90,7 @@ The complexity of this plugin overwhelmingly involves hacking around with yum, w
 
 # Version History
 
+* 1.0.2 - Fix typos in yaml and help
```

Comment:
We don't need this since this hasn't been released since last Hub changes. No version bump here.

Reply:
@mhofert-r7 Still need to address this one.

Reply:
We can remove this line